api/types/volumes-generated.go

```diff
@@ -0,0 +1,53 @@
+// Code generated by swagger-gen. DO NOT EDIT
+
+package types
+
+type Volume struct {
+	// Date/Time the volume was created.
+	CreatedAt string `json:"CreatedAt,omitempty"`
+	// Name of the volume driver used by the volume.
+	Driver string `json:"Driver"`
+	// User-defined key/value metadata.
+	Labels map[string]string `json:"Labels"`
+	// Mount path of the volume on the host.
+	Mountpoint string `json:"Mountpoint"`
+	// Name of the volume.
+	Name string `json:"Name"`
+	// The driver specific options used when creating the volume.
+	Options map[string]string `json:"Options"`
+	// The level at which the volume exists. Either `global` for cluster-wide, or `local` for machine level.
+	Scope string `json:"Scope"`
+	/*
+	   Low-level details about the volume, provided by the volume driver.
+	   Details are returned as a map with key/value pairs:
+	   `{"key":"value","key2":"value2"}`.
+
+	   The `Status` field is optional, and is omitted if the volume driver
+	   does not support this feature.
+	*/
```

Comment:
Not a big fan of the block-style comments; seems they're generated by this https://github.com/dave/jennifer/blob/master/jen/comments.go#L85-L93, so not sure there's an easy workaround (other than splitting on newlines and appending a single-line comment for each line)

Reply:
Yeah if there's a newline in a comment, jennifer will use multi-line syntax. However, if you prefix your comment with `//` it'll be rendered as-is. You'd need to manually add the `//` to the start of every line though. Feel free to ping me if you'd like any more jennifer help!